compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -513,9 +515,10 @@ class MapAllocatorCache {
         Quarantine[I].invalidate();
       }
     }
+    QuarantinePos = -1U;
```

Comment:
just a minor question, why do we need to reorder this? I think it's guarded by the same lock as other cache data members.